spec/adapters/signal_adapter/outbound_spec.rb

```diff
@@ -77,8 +78,13 @@
       end
 
       describe 'on error' do
-        before { stub_request(:post, 'http://signal:8080/v2/send').to_return(status: 400) }
-        it { should raise_error(Net::HTTPServerException) }
+        before(:each) { stub_request(:post, 'http://signal:8080/v2/send').to_return(status: 400) }
+
+        it 'ErrorNotifier' do
```

Comment:
Oh, maybe change this to something that sounds a little more natural?

```suggestion
        it 'reports the error' do
```